src/axom/core/MapCollection.hpp

```diff
@@ -179,8 +179,15 @@ class MapCollection : public ItemCollection<T>
   ///
   bool hasItem(const std::string& name) const
   {
-    typename MapType::const_iterator mit = m_name2idx_map.find(name);
-    return (mit != m_name2idx_map.end() ? true : false);
+    if(m_name2idx_map.empty())
```

Comment:
These `empty()` checks prevent calling `m_name2idx_map.find()` and `m_name2idx_map.end()` on an empty hash table. Sparsehash assumes that we have set sentinel values for empty and deleted entries, and `MapCollection` only (currently) does this when inserting or deleting entries.